src/main/java/atmosfears/AtmosFearsBE/database/Particulate.java

```diff
@@ -0,0 +1,25 @@
+package atmosfears.AtmosFearsBE.database;
+
+import lombok.Getter;
+
+public enum Particulate {
+
+    // TODO: Add correct units for each particulate
+    CO("mg/m3"),
+    NO2("µg/m3"),
+    O3("µg/m3"),
+    PM10("µg/m3"),
+    PM25("µg/m3"),
+    SO2("µg/m3");
```

Comment:
I believe we don't have these units? or did you forget about this TODO?

Reply:
I use it in `TimeSeriesController`. This unit is displayed on the `TimeSeries` chart.